README.md

```diff
@@ -13,7 +13,11 @@ The Vulcan Academy's operating system.
 1. All rules and resolutions should be logically self-consistent.
 1. Vulcans declare [Crocker's Rules](https://wiki.lesswrong.com/wiki/Crocker's_rules) with each other.
 
+## Conventions
+
+1. The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in these rules are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
 ## Contents
 
 1. [Lexicon](Lexicon.md)
-1. [Resources](Resources.md)
\ No newline at end of file
+1. [Resources](Resources.md)
```

Comment:
Are the line endings here the same?

Reply:
They should be, not sure why it shows up in the diff